### ISSUE TITLE
Playbooks UI Upgrades

### ISSUE THREAD

**Issue by peth-yursick** [MetaFam/metagame-wiki#257]
Playbooks are short onboarding & skill-building guides for the DAO space.

[Playbooks on the wiki](https://wiki.metagame.wtf/docs/playbooks/browse)
[Playbook WIPs on Notion](https://www.notion.so/Write-a-Playbook-3d8b963053f5457091abfe74f06248c9)
[📕-playbooks discussion on Discord](https://discord.gg/TCFdHCr)
[Playbooks UI WIP](https://www.figma.com/file/EDj3DFVrV2ms6QA7Q8gRdp/Playbooks-UI---Quick-Questions?node-id=0%3A1)

Useful things:
- Categories & filtering
- Upvoting/downvoting playbooks
- Time to read
- Reacting to playbooks
- Comment
- Share
- Fork
- Bookmark (pin to your "dashboard" or "goals" both of which are nonexistant features)

Being designed by @FearlessThompson & @davort

**Comment by FearlessThompson** [comment]
Additional features maybe - Voice Narration & Gated Content (NFTs or SEEDs)
Categories: DAOs, DeFi, NFTs & dApps

Any other categories, are we going to purely crypto focused? 

If we could also add notification, if you Follow a particular playbook author.

Thoughts?

**Comment by davort** [comment]
This is awesome. Thank you @FearlessThompson and @peth-yursick.
We now have enough ideas to start planning 👍 The method I'd propose is **user story mapping**. 

But before we got to that, it would be ideal if we could get a builder to work on it with us. 
Hopefully this person will also know what's possible with the new platform that will be underpinning the Wiki (Docusaurus?), and thus the Playbooks UI. All in the interest of keeping us as efficient as possible.

We (designers) can of course run alone for now, but it would be at a much higher risk of running in the wrong direction. How should we proceed, @peth-yursick?

**Comment by FearlessThompson** [comment]
No worries! Understood that would be ideal, I've been meaning to reach out to @jonathanprozzi too. He's been assigned, so hopefully we can get this going. 

How would we like to approach the User Story Mapping,I was thinking of maybe a Follow Author feature on Iteration 3. 

Depending on availability, would the 3 of us need a quick meeting to discuss the next steps? Yourself & @jonathanprozzi this week?

**Comment by jonathanprozzi** [comment]
Still digging in to this, but a few of the lower lifts can be integrated right now (such as the _reading time_) -- will keep exploring and will ask any questions!

**Comment by davort** [comment]
@FearlessThompson - ideally, we'd know all that is, and isn't possible. But even if we don't know (yet), we can still plan, just somewhat more conservatively. 
Even then, we could plan for a _solid first release_, which would be way better from what we've had thus far.

Follow Author - great idea, keep them coming! 😊

@jonathanprozzi - thanks for the update on the "reading time". It will be a great feature to have in the first release! Please keep us posted when you learn any new info. I'd for one be very interested in knowing whether we can enable peoe to "add books to a reading list", make bookmarks, add notes, etc. Just as an idea.

**Comment by davort** [comment]
**Playbooks UI - User Story Mapping session**

@FearlessThompson and I met with @luxumbra and @jonathanprozzi yesterday, to take a stab at making a plan for building the Playbooks UI. 

We used User Story Mapping as our method of choice, because it let us put all our thoughts onto the [FigJam board](https://www.figma.com/file/lFi0dlWGx2KjHLhQJ1TyNa/Playbooks-User-story-mapping?node-id=0%3A1), which allowed us to make room in our minds for _even better ideas_ :slight_smile: Once on the FigJam board, we could take a look at these ideas, and establish what we did, and didn't know, was feasible or even possible. 

![CleanShot 2021-07-09 at 21 26 15](https://user-images.githubusercontent.com/1573304/125127145-6709ad80-e0fc-11eb-8a1a-fb8c6549aa6f.png)

This led us to making following conclusions:

1. The initial idea was to make the Playbooks UI a part of the revamped Wiki, which will be built with Docusaurus. The more we were thinking about how to make a truly useful UI for enjoying Playbooks, the more it became clear that many of the features would rely on the ability for players to be logged into the system, which Docusaurus isn't really meant to support out-of-the-box. Hence, the first/interim release will most likely still run as part of Wiki, but it will be limited in features, because of Docusaurus' limitations. 
2. The second, and all subsequent releases, will most likely have to be built as a part of MyMeta. The Quests UI, which itself is still pending finalisation, seems to offer a good foundation for the future Playbooks UI. This is the idea that we need to work on more thoroughly moving forward. The idea is appealing, since it would allow us to reuse something we've already built. Thus, we'd save time, and maintain a coherent UX :raised_hands:
3. There are a few open questions, such as how do we deal with categories, tags, etc.? @FearlessThompson is on top of that one.

We'll meet again after the weekend, with new findings, so that we can hopefully lock in the initial release. The result of this will be a list of features to consider while designing the UI.

Any questions about any of this, please let me know.

**Comment by davort** [comment]
Here's the first batch of mobile wireframes. The goal was to nail down the basic layout for mobile, while making sure all is accessible in terms of sizes for tap target areas, font-sizes and colors.

The mockups are showing:

1. Landing page view → _a sort of a digest, showing the latest—or most read?—additions to each category_
2. Landing page w/ category hidden/collapsed → _just an idea for keeping mobile view shorter and easier to scroll, in case the user isn't interested in certain categories_
3. Category view
4. Article view
5. Author view → _should we add a 2-3 line long "bio" underneath the username?_

![CleanShot 2021-07-21 at 09 57 34](https://user-images.githubusercontent.com/1573304/126452947-a1a96e06-03f7-443f-ad46-8fa80f863ed9.png)

Eager to hear any and all of your comments or questions!

**Comment by luxumbra** [comment]
Loving the progress on this. Apologies for the slow response @davort 

> 1. Landing page view → _a sort of a digest, showing the latest—or most read?—additions to each category_

Great idea @davort 👍🏻 Sort by latest or most read would be nice. 

> 2. Landing page w/ category hidden/collapsed → _just an idea for keeping mobile view shorter and easier to scroll, in case the user isn't interested in certain categories_

Nice. maybe show the top/newest post and a 'view more books' to expand.

> 3. Category view

No comment. 👍🏻  

> 4. Article view

No comment 💪🏻  

> 5. Author view → _should we add a 2-3 line long "bio" underneath the username?_

Yes, I think a short bio would be great. or link to their mymeta profile.

**Comment by Baconaetor** [comment]
My understanding was that this UI design was for My Meta/Meta OS. Is this the case, or are we designing it for the wiki?

**Comment by davort** [comment]
> My understanding was that this UI design was for My Meta/Meta OS.

This is correct. You can find more context in the [Playbooks UI Notion](https://skinny-estimate-0fd.notion.site/Playbooks-UI-5ae958cfe1e945658deecd3a78764a0a) page.

**Comment by Baconaetor** [comment]
Ok, is there a way to move this issue to over there then?
